RosBridgeClient/Message.cs

```diff
@@ -15,6 +15,7 @@
 limitations under the License.
 */
 
```

Comment:
Please add modification comment:
/#
added classes GeomeryAccel, SensorJoy and GeometryPoseStamped
(C) <your company, your name (optionally your mail)>, 2017
*/